site/app/controllers/student/GradeInquiryController.php

```diff
@@ -192,7 +207,7 @@ public function getSingleGradeInquiryPost($gradeable_id) {
     /**
      * @param $gradeable_id
      * @Route("/courses/{_semester}/{_course}/gradeable/{gradeable_id}/grade_inquiry/toggle_status", methods={"POST"})
-     * @return MultiResponse|null null is for tryGetGradeable and tryGetGradedGradeable
+     * @return MultiResponse|JsonResponse|null null is for tryGetGradeable and tryGetGradedGradeable
```

Comment:
Is there any reason why this can't be refactored slightly to only return `JsonResponse|null`?